plutus-ledger-api/src/PlutusLedgerApi/V2/ParamName.hs

```diff
@@ -200,5 +202,117 @@ data ParamName =
+  | Bls12_381_G2_compress'memory'arguments
+  | Bls12_381_G2_equal'cpu'arguments
+  | Bls12_381_G2_equal'memory'arguments
+  | Bls12_381_G2_hashToGroup'cpu'arguments'intercept
+  | Bls12_381_G2_hashToGroup'cpu'arguments'slope
+  | Bls12_381_G2_hashToGroup'memory'arguments
+  | Bls12_381_G2_neg'cpu'arguments
+  | Bls12_381_G2_neg'memory'arguments
+  | Bls12_381_G2_scalarMul'cpu'arguments'intercept
+  | Bls12_381_G2_scalarMul'cpu'arguments'slope
+  | Bls12_381_G2_scalarMul'memory'arguments
+  | Bls12_381_G2_uncompress'cpu'arguments
+  | Bls12_381_G2_uncompress'memory'arguments
+  | Bls12_381_finalVerify'cpu'arguments
+  | Bls12_381_finalVerify'memory'arguments
+  | Bls12_381_millerLoop'cpu'arguments
+  | Bls12_381_millerLoop'memory'arguments
+  | Bls12_381_mulMlResult'cpu'arguments
+  | Bls12_381_mulMlResult'memory'arguments
+  | Keccak_256'cpu'arguments'intercept
+  | Keccak_256'cpu'arguments'slope
+  | Keccak_256'memory'arguments
+  | Blake2b_224'cpu'arguments'intercept
+  | Blake2b_224'cpu'arguments'slope
+  | Blake2b_224'memory'arguments
+  | AndByteString'cpu'arguments'intercept
+  | AndByteString'cpu'arguments'slope1
+  | AndByteString'cpu'arguments'slope2
+  | AndByteString'memory'arguments'intercept
+  | AndByteString'memory'arguments'slope
+  | OrByteString'cpu'arguments'intercept
+  | OrByteString'cpu'arguments'slope1
+  | OrByteString'cpu'arguments'slope2
+  | OrByteString'memory'arguments'intercept
+  | OrByteString'memory'arguments'slope
+  | XorByteString'cpu'arguments'intercept
+  | XorByteString'cpu'arguments'slope1
+  | XorByteString'cpu'arguments'slope2
+  | XorByteString'memory'arguments'intercept
+  | XorByteString'memory'arguments'slope
+  | ComplementByteString'cpu'arguments'intercept
+  | ComplementByteString'cpu'arguments'slope
+  | ComplementByteString'memory'arguments'intercept
+  | ComplementByteString'memory'arguments'slope
+  | ReadBit'cpu'arguments
+  | ReadBit'memory'arguments
+  | WriteBits'cpu'arguments'intercept
+  | WriteBits'cpu'arguments'slope
+  | WriteBits'memory'arguments'intercept
+  | WriteBits'memory'arguments'slope
+  | ReplicateByte'cpu'arguments'intercept
+  | ReplicateByte'cpu'arguments'slope
+  | ReplicateByte'memory'arguments'intercept
+  | ReplicateByte'memory'arguments'slope
+  | ShiftByteString'cpu'arguments'intercept
+  | ShiftByteString'cpu'arguments'slope
+  | ShiftByteString'memory'arguments'intercept
+  | ShiftByteString'memory'arguments'slope
+  | RotateByteString'cpu'arguments'intercept
+  | RotateByteString'cpu'arguments'slope
+  | RotateByteString'memory'arguments'intercept
+  | RotateByteString'memory'arguments'slope
+  | CountSetBits'cpu'arguments'intercept
+  | CountSetBits'cpu'arguments'slope
+  | CountSetBits'memory'arguments
+  | FindFirstSetBit'cpu'arguments'intercept
+  | FindFirstSetBit'cpu'arguments'slope
+  | FindFirstSetBit'memory'arguments
+  | Ripemd_160'cpu'arguments'intercept
+  | Ripemd_160'cpu'arguments'slope
+  | Ripemd_160'memory'arguments
+  -- Not yet deployed
```

Comment:
Aren't these also to be deployed in PV11? Why are they "Not yet deployed"?

Reply:
I've changed that to "To be deployed in PV11", but I can take it out if you want.  I may have copied it from the PlutusV3 parameters, where it marks the boundaries between the parameters for the things that are on the chain now and those that aren't.  

A lot of the code in plutus-ledger-api is a bit weird because in a sort of in-between state where it's about stuff that's waiting to happen.